groovy/postInit/chemistry/organic_chemistry/petrochemistry/oil_refining/LighterEnds.groovy

```diff
@@ -0,0 +1,152 @@
+import petrochemistry.Petrochemistry
+
+DT = recipemap('distillation_tower')
+BCR = recipemap('bubble_column_reactor')
+
+// Debutanization
+
+DT.recipeBuilder()
+    .fluidInputs(fractions.naphtha.getCrude(10000))
+    .fluidOutputs(fluid('sulfuric_debutanized_naphtha') * 8000)
+    .fluidOutputs(fluid('sulfuric_fuel_gas') * 2000)
+    .duration(400)
+    .EUt(30)
+    .buildAndRegister()
+
+DT.recipeBuilder()
```

Comment:
This seems to be a duplicate with the recipe 10 lines after this, if I'm reading what getting the basic fraction type means.

Reply:
nah this gets from crude to sulfuric, next recipe partitions the naphtha into heavy and light components